crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -661,7 +661,14 @@ where
         self.emit_leading_comments_of_span(v.span, false)?;
 
         if self.cfg.minify {
-            self.wr.write_lit(v.span, &v.value.to_string())?;
+            let value = if v.value >= 10000000000000000_i64.into() {
```

Comment:
What's this value?

Reply:
Where does it come from?

Reply:
Oh it's not a change request. It was just question

Reply:
10000000000000000n toString ->
10000000000000000n (length 17)
10000000000000000n toString(16) ->
0x2386f26fc10000n (length 16)

So I ran a program to find the smallest bigint that hex representation is shorter than decimal and the result is 10000000000000000n

Reply:
Got it, thank you!